lib/modules/dosomething/dosomething_reportback/dosomething_reportback.reportback_transformer.inc

```diff
@@ -9,6 +9,77 @@ abstract class ReportbackTransformer {
 
 
   /**
+   * Get the URI for the next page in the collection of data.
+   *
+   * @param $data
+   * @param $parameters
+   * @param $endpoint
```

Comment:
You missed a newline between `@param` and `@return`. Pls fix here and everywhere below.